### DISCUSSION TITLE
Theme and Google Translate

### DISCUSSION THREAD

**Discussion by erichkist** [adityatelange/hugo-PaperMod#334] [Q&A]
Hey folks,

I'm testing hugo using PerperMod theme and I realize that Google Translate can't work with Theme. It seems that CSS is loaded but it didn't apply.

https://translate.google.com/translate?sl=en&tl=be&u=https://adityatelange.github.io/hugo-PaperMod/

Any idea?

**Comment by adityatelange** [comment]
Have you checked `console` tab in DevTools ?

**Reply by erichkist** [reply]
> Have you checked `console` tab in DevTools ?

`Subresource Integrity: The resource 'https://adityatelange.github.io/hugo-PaperMod/assets/css/stylesheet.min.d85457619e67d31b97f40ed67c71b5efe0b52508ba759e5a9da0760afd6707db.css' has an integrity attribute, but the resource requires the request to be CORS enabled to check the integrity, and it is not. The resource has been blocked because the integrity cannot be enforced.`

<img width="1791" alt="Screen Shot 2021-03-29 at 4 26 29 PM" src="https://user-images.githubusercontent.com/130042/112889197-b545f400-90ab-11eb-8fa6-d47e9260e7d3.png">

**Reply by erichkist** [reply]
Ok. I was checking CORS + integrity and it's necessary to explicit set the attribute `crossorigin` as `anonymous` or `use-credentials`. In the Google Translate case, I applied anonymous and it works.
